docs/release-notes/2.16.0.md

```diff
@@ -0,0 +1,138 @@
+# Percona Monitoring and Management 2.16.0
+
+<table class="docutils field-list" frame="void" rules="none">
+  <colgroup>
+    <col class="field-name">
+    <col class="field-body">
+  </colgroup>
+  <tbody valign="top">
+    <tr class="field-odd field">
+      <th class="field-name">Date:</th>
+      <td class="field-body">April 12, 2021</td>
+    </tr>
+    <tr class="field-even field">
+      <th class="field-name">Installation:</th>
+      <td class="field-body">
+        <a class="reference external" href="https://www.percona.com/software/pmm/quickstart">Installing Percona Monitoring and Management</a></td>
+    </tr>
+  </tbody>
+</table>
+
+Percona Monitoring and Management (PMM) is a free and open-source platform for managing and monitoring MySQL, MongoDB, and PostgreSQL performance.
+
+## Release Highlights
+
+* **Amazon RDS PostgreSQL monitoring**
+
+    AWS monitoring in PMM now covers PostgreSQL RDS and PostgreSQL Aurora types. PMM will include them in [Discovery UI](../setting-up/client/aws.md#adding-an-amazon-rds-postgresql-instance) where they can be added which will result in node related metrics as well as PostgreSQL database performance metrics. Before this release, this was available only to MySQL-related instances from Amazon RDS.
+
+* **Support for pg_stat_monitor 0.8**
+
+    Added compatibility with [pg_stat_monitor plugin v 0.8.0](https://github.com/percona/pg_stat_monitor/releases/tag/REL0_8_0_STABLE ). This is not exposing the new features for the plugin in PMM yet, but ensures Query Analytics metrics are collected to the same degree it was with version 0.6.0 of the plugin.
+
+* **Consistent support of Technical Preview Features**
+
+    Reworked the PMM Settings page to make it clear what features are in Technical Preview vs General Availability (GA) and to simplify activation/deactivation of technical preview features. We also provide a better definition of what a [Technical Preview](../details/glossary.md#technical-preview) is.
+
+* **Make non-dashboard pages part of grafana fork codebase (migration)**
+
+* **Security Threat Tool Scheduling - Manage execution and execution intervals**
+
+    Security Threat Tool users are now able to control the Security Check execution time intervals for groups of checks and move checks between groups, and disable individual checks if necessary.
+
+* **Azure Discovery and Node metrics extraction**
+
+    **Technical Preview**: PMM will have the same level of support for Microsoft Azure Database as a Service (DBaaS) as we have for AWS's DBaaS (RDS/Aurora on MySQL or Postgres). You will be able to easily discover and add Azure databases for monitoring by PMM complete with node-level monitoring. This feature is available only if you explicitly turn it ON on the Settings page. Turning this feature OFF will not remove added Services from monitoring, it will just hide the ability to Discover and add new Microsoft Azure Services.
```

Comment:
```suggestion
    **Technical Preview**: PMM will have the same level of support for Microsoft Azure Database as a Service (DBaaS) as we have for AWS's DBaaS (RDS/Aurora on MySQL or Postgres). You will be able to easily discover and add Azure databases for monitoring by PMM complete with node-level monitoring. This feature is available only if you explicitly turn it ON on the Settings page. Turning this feature OFF will not remove added Services from monitoring, it will just hide the ability to Discover and add new Microsoft Azure Services.
    
    This feature technical preview because we releasing it as soon as possible to get some feedback from users. We are expecting to do more work on this feature to make it more API and resource-efficient. 
```